include/fmt/chrono.h

```diff
@@ -1135,10 +1155,18 @@ template <typename OutputIt, typename Char> class tm_writer {
   }
 
  public:
-  tm_writer(const std::locale& loc, OutputIt out, const std::tm& tm)
+  tm_ss_writer(const std::locale& loc, OutputIt out, const tm_ss& tm_ss_val)
       : loc_(loc),
         is_classic_(loc_ == get_classic_locale()),
         out_(out),
+        tm_ss_(tm_ss_val),
+        tm_(tm_ss_val.tm) {}
+
+  tm_ss_writer(const std::locale& loc, OutputIt out, const std::tm& tm)
+      : loc_(loc),
+        is_classic_(loc_ == get_classic_locale()),
+        out_(out),
+        tm_ss_(empty_tm_ss::instance().get_tm_ss()),
         tm_(tm) {}
```

Comment:
Similarly, there should be one ctor that takes an additional subseconds argument defaulted to 0.

Reply:
I don't think it will work with an additional argument defaulted to 0. There is a difference between a timestamp based on `std::chrono::seconds(1)` (printed as "....01") and `std::chrono::milliseconds(1000)` (printed as "01.000" - the MS implementation of C++20 std::format does the same). Cf. the last unit test with `EXPECT_EQ(fmt::format("{:%S}", t10), "02.000");`

So I made two constructors and one sets `hasSubsecs` to true.

Reply:
Used pointer now